docs/src/pages/Conformance.md

```diff
@@ -114,8 +114,13 @@ In addition, the following search parameters are supported on all resources:
 * `_has`
 
 These parameters can be used while searching any single resource type or while searching across resource types (whole system search).
-The `_type` parameter is special in that it is only applicable for whole system search. The `_has` parameter has two restrictions:
-* `_has` cannot be used with whole system search.
+
+The `_type` parameter has two restrictions:
+* It may only be used with whole system search.
+* It may only be specified once in a search.
```

Comment:
The HTTP Headers section does mention how lenient factors in, so I think that's good. For this comment, I updated these specific parameter sections to mention that only the first occurrence it used in lenient mode, since that is specific behavior to these parameters.